src/setup.cfg.m4

```diff
@@ -107,27 +107,26 @@ sage.repl.rich_output =
 
 sage =
     ext_data/*
-    ext_data/kenzo/*
```

Comment:
I would suggest to back out the cosmetic change (alphabetic sorting) from this PR because it will conflict with #36951